### PR TITLE
add rust crate downloads

### PR DESCRIPTION
**Pull request by chirag-jn**

### CODE REVIEW

**Review by prrao87 — APPROVED** [review]
Awesome feature!

File: docs/tools/vdb_table/vendor.schema.json [hunk 1]
```diff
@@ -38,7 +38,8 @@
     "github_stars": {"allOf": [{"$ref": "#/$defs/statsWithSource"}], "$comment": "Stats | GitHub ⭐ | The total number of stars for the core product repository." },
     "docker_pulls": {"allOf": [{"$ref": "#/$defs/statsWithSource"}],"$comment": "Stats | Docker Pulls | The total number of pulls for the core product's docker image." },
     "pypi_downloads": {"allOf": [{"$ref": "#/$defs/statsWithSource"}], "$comment": "Stats | PyPi Downloads | The number of downloads (in last 90 days) for the core product's python package." },
-    "npm_downloads": {"allOf": [{"$ref": "#/$defs/statsWithSource"}], "$comment": "Stats | NPM Downloads | The number of downloads (in last 90 days) for the core product's NPM package." }
+    "npm_downloads": {"allOf": [{"$ref": "#/$defs/statsWithSource"}], "$comment": "Stats | NPM Downloads | The number of downloads (in last 90 days) for the core product's NPM package." },
+    "rust_downloads": {"allOf": [{"$ref": "#/$defs/statsWithSource"}], "$comment": "Stats | Rust Crates Downloads | The number of downloads (in last 90 days) for the core product's Rust crate." }
```
Comment (prrao87) [inline docs/tools/vdb_table/vendor.schema.json:42]: @AruneshSingh If we're referring to the other two as `pypi_downloads` and `npm_downloads` downloads, should this be renamed as `crates_io_downloads` to refer to Rust's [crates.io](https://crates.io/)? That way we're consistently referring to the package manager names and not the programming language.

Comment (prrao87) [inline docs/tools/vdb_table/vendor.schema.json:42]:
```suggestion
    "crates_io_downloads": {"allOf": [{"$ref": "#/$defs/statsWithSource"}], "$comment": "Stats | Rust Crate Downloads | The number of downloads (in last 90 days) for the core product's Rust crate." }
```

Comment (prrao87) [reply docs/tools/vdb_table/vendor.schema.json:42]: I just also realized that `rust_downloads` refers to the variable name in each of the JSON files, so you're free to decide whether it's worth changing for consistency reasons.

**Comment by chirag-jn** [comment]
@prrao87 sounds about right. I'll make the change.

**Comment by chirag-jn** [comment]
> @prrao87 sounds about right. I'll make the change.

done.

**Comment by AruneshSingh** [comment]
Looks good to me, will merge in the morning after doing the required changes on the frontend.

**Pull request merged**